labgrid/driver/commandmixin.py

```diff
@@ -21,4 +21,22 @@ def wait_for(self, cmd, pattern, timeout=30.0, sleepduration=1):
         while not any(pattern in s for s in self.run_check(cmd)) and not timeout.expired:
             sleep(sleepduration)
         if timeout.expired:
-            raise ExecutionError("Wait timeout expired")
+            raise ExecutionError("Wait timeout expired", '')
+
+    @Driver.check_active
+    @step(args=['cmd'], result=True)
+    def run_check(self, cmd: str, timeout=30):
```

Comment:
This removes the _run_check method, which can be used internally during activation and will break _put_ssh_key of the ShellDriver.